diff --git a/configure b/configure
index d461294..5726399 100644
--- a/configure
+++ b/configure
@@ -414,7 +414,7 @@ if (@ngx_ld_opts) {
     $ld_opts = " \\\n  --with-ld-opt='@ngx_ld_opts'";
 }
 
-my $cmd = "sh ./configure --prefix=$ngx_prefix"
+my $cmd = "sh ./auto/configure --prefix=$ngx_prefix"
     . $resty_opts
     . $ld_opts
     . (@ngx_opts ? " \\\n  " . quote_cli_args(\@ngx_opts) : "");
